deblur/deblurring.py

```diff
@@ -36,7 +36,10 @@ def get_sequences(input_seqs):
     seqs = [Sequence(label, seq) for label, seq in input_seqs]
 
     if len(seqs) == 0:
-        raise ValueError("No sequences found!")
+        logger = logging.getLogger(__name__)
+        logger.warn('No sequences found in fasta file!')
```

Comment:
Does the thread write the name of the file that is processing when it starts processing it? That may be enough to help to troubleshoot.